### PR TITLE
spelling: methods

### PR DESCRIPTION
**Pull request by jsoref**
Spelling fixes for methods

## Description
Semi-automated spelling review of codebase

## Motivation and Context
I have a toolchain which I've been evolving for a while, and I like to test it on projects, especially as a way to give a little in appreciation of projects that I may be using/have used/plan to use.

## How Has This Been Tested?
- [ ] AppVeyor and Travis built this code

## Types of change
- [x] methods

## Checklist:

- [x] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
Jenkins build this please

**Comment by MartijnKaijser** [comment]
@jsoref can you fix android as well cause i fails in the JNI part

**Comment by garbear** [comment]
is https://github.com/koying/libdroidjni an upstream? they have this misspelling too

**Comment by jsoref** [comment]
@MartijnKaijser: I can't figure out what android's trying to tell me,

**Comment by MartijnKaijser** [comment]
`In file included from /home/jenkins/workspace/Android-X86/xbmc/platform/android/jni/List.cpp:29:0:
/home/jenkins/workspace/Android-X86/xbmc/platform/android/jni/InetAddress.h:50:38: error: conflicting declaration 'typedef class std::vector<CJNIInetAddress> CJNIInetAddress'
 typedef std::vector<CJNIInetAddress> CJNIInetAddress;
                                      ^
/home/jenkins/workspace/Android-X86/xbmc/platform/android/jni/InetAddress.h:26:7: note: previous declaration as 'class CJNIInetAddress'
 class CJNIInetAddress : public CJNIBase
       ^
build/android_jni/CMakeFiles/platform_android_jni.dir/build.make:830: recipe for target 'build/android_jni/CMakeFiles/platform_android_jni.dir/List.cpp.o' failed`

File: xbmc/platform/android/jni/InetAddress.h [hunk 1]
```diff
@@ -47,4 +47,4 @@ class CJNIInetAddress : public CJNIBase
   static const char *m_classname;
 };
 
-typedef std::vector<CJNIInetAddress> CJNIInetAddresss;
+typedef std::vector<CJNIInetAddress> CJNIInetAddress;
```
Comment (jsoref) [inline xbmc/platform/android/jni/InetAddress.h]: This should be CJNIInetAddresses

**Comment by jsoref** [comment]
Ah, finally got it.

**Comment by Rechi** [comment]
Jenkins build this please

**Comment by Rechi** [comment]
Jenkins build this please

**Pull request merged**